libs/common/src/middleware/create-validation-middleware-for-schema.ts

```diff
@@ -1,7 +1,8 @@
 import { ZodError, ZodType } from 'zod';
 import { NextFunction, Request, Response } from 'express';
 import { HttpStatusCode } from 'axios';
-import { AnyObject, API_ERROR_CODE, ApiErrorCode, ApiErrorResponseBody } from '@ukef/dtfs2-common';
+import { AnyObject, ApiErrorCode, ApiErrorResponseBody } from '../types';
+import { API_ERROR_CODE } from '../constants';
```

Comment:
Whats the reason for importing from local constants instead of the common library please?

Reply:
This is a file in libs/common

Reply:
Sorry I did not realise 😂 🤦